.github/workflows/coverage.yml

```diff
@@ -3,6 +3,7 @@ name: Coverage
 on:
   push:
     branches:
+      - dev
```

Comment:
Maybe worthwhile considering restricting to actual code containing folders, because its a waste of resources to run on mere fixes to README etc.

Reply:
Good idea – I'll add a section to disable running on certain paths.